src/rpc/rpc_ws.js

```diff
@@ -57,7 +58,7 @@ class RpcWsConnection extends RpcBaseConnection {
             // may send fake http messages inside the websocket
             // in order to poison intermediate proxy caches.
             // reduces rpc throughput to ~70 MB/s
-            mask: false,
+            // mask: false,
```

Comment:
?

Reply:
we are enabling the ws mask now as the websockets is closing the connection if the mask is set.
https://github.com/websockets/ws/commit/4f293a8726092c75539287dd07358afaf151a2e5

Reply:
@liranmauda can you test the performance of rpc_benchmark over websocket with masking?
See the comment above the mask saying that without it we experienced a very low throughput of 70MB/sec.
you need to run one server and one client like this:
```
node src/rpc/rpc_benchmark.js --server --proto wss
node src/rpc/rpc_benchmark.js --client --proto wss
```

Here are the results I got with node 12 and older libs when toggling masking:
```
{ nodejs: 'v12.8.2', ws: '7.2.1', proto: 'wss', mask: false } => 500 MB/sec
{ nodejs: 'v12.8.2', ws: '7.2.1', proto: 'wss', mask: true } => 400 MB/sec
```

with the ws protocol (no TLS):
```
{ nodejs: 'v12.8.2', ws: '7.2.1', proto: 'ws', mask: false } => 650 MB/sec
{ nodejs: 'v12.8.2', ws: '7.2.1', proto: 'ws', mask: true } => 550 MB/sec
```

can you test these 4 cases too on node 14 and new ws?
seems like the effect is not as drastic as it was before.

Reply:
Two more points:
1. We can explore is if we can make mask:false work on the new we library version. We just failed to run it but it might be possible.
2. We can quite easily change our RPC to use the `tls:` protocol instead, in which we are doing the simple message framing on the socket, this is already being used for n2n connections so this is very stable. See [here](https://github.com/noobaa/noobaa-core/blob/50afd6ee7e161954aad2590dc66c2888432f6efc/src/rpc/rpc.js#L526-L530)